### ISSUE TITLE
Question about the calculation of urf depth loss

### ISSUE THREAD

**Issue by meneldil12555** [nerfstudio-project/nerfstudio#1871]
If my understandings are correct, the nerf weights (transmittance * (1-exp(sigma*delta)) should approximately sum up to 1.
But in the implementation of urban_radiance_field_depth_loss(), the line_of_sight_loss_near seems to be optimizing weights towards the probability density of a normal distribution, which could be way more than 1.
`target_distribution = torch.distributions.normal.Normal(0.0, sigma / URF_SIGMA_SCALE_FACTOR)
line_of_sight_loss_near = (weights - torch.exp(target_distribution.log_prob(steps - termination_depth))) ** 2`
It doesn't matter if the rendering equation is written in integral form (as in urf's original paper), because sigma(density) can be very high. But I didn't find how the paper discretized the equation. Did I miss something? I guess the correct discretization may be to optimize weights toward the cumulative distribution function.

**Comment by YZsZY** [comment]
you are right, the distribution should be normalized to make it sum to 1, just follow this issue:https://github.com/nerfstudio-project/nerfstudio/issues/1793
